docker/Dockerfile

```diff
@@ -11,6 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+FROM centos:centos7 AS dist
+ARG PRESTO_VERSION
+ARG CLIENT_VERSION=${PRESTO_VERSION}
+COPY presto-cli-${CLIENT_VERSION}-executable.jar /usr/bin/presto
+COPY presto-server-${PRESTO_VERSION} /usr/lib/presto
+RUN hardlink -cf /usr/lib/presto
```

Comment:
do we have any initially empty files that need to be kept unlinked?

(i think we don't have, but worth checking)

Reply:
Just checked, no empty files